src/components/forms/Form/ComplexForm.stories.tsx

```diff
@@ -32,6 +32,41 @@ export default {
   parameters: { controls: { exclude: baseProps } },
 };
 
+const UnknownSubmitErrorTemplate: StoryFn<typeof Form> = (args) => {
+  const [form] = Form.useForm();
+
+  return (
+    <Form
+      form={form}
+      {...args}
+      onSubmit={(v) => {
+        console.log('onSubmit:', v);
```

Comment:
Please, don't use `console.log` to print out a callback's output, use `@storybook/actions` instead.

If your callback matches the default regex `on*`, the storybook will generate actions automatically. So in your case, it's enough to replace `console.log()` with `args.onSubmit(v)`:

```diff
onSubmit={(v) => {
- console.log('onSubmit:', v);
+ args.onSubmit(v);
}}
```

Reply:
It's not a matter of this PR. But I can do it separately.

Reply:
It just FYI